Documentation/deploy-master.md

```diff
@@ -1,676 +1,88 @@
-            # access, configure the container's /etc/hosts to resolve
-            # kubernetes.default to the correct service clusterIP.
-            - name: CONFIGURE_ETC_HOSTS
-              value: "true"
 ```
-
-## Start Services
-
-Now that we've defined all of our units and written our TLS certificates to disk, we're ready to start the master components.
-
-### Load Changed Units
-
-First, we need to tell systemd that we've changed units on disk and it needs to rescan everything:
-
-```sh
-$ sudo systemctl daemon-reload
+$ kubectl get nodes -o wide
+i-0e475be181f7e1973 10.0.19.110 52.35.157.146
+i-03d1bc5e86b7e8741 10.0.2.146 34.209.11.147
+i-038ea56416abdd140 10.0.36.204 34.210.154.200
+i-024d0aa51334a068b 10.0.77.101
 ```
 
-### Configure flannel Network
-
-Earlier it was mentioned that flannel stores cluster-level configuration in etcd. We need to configure our Pod network IP range now. Since etcd was started earlier, we can set this now. If you don't have etcd running, start it now.
+By default only the master Nodes, the machines running the Kubernetes API server, will have external IPs. Choose one of the public IPs from the third column and login.
 
-* Replace `$POD_NETWORK`
-* Replace `$ETCD_SERVER` with one url (`http://ip:port`) from `$ETCD_ENDPOINTS`
-
-```sh
-$ curl -X PUT -d "value={\"Network\":\"$POD_NETWORK\",\"Backend\":{\"Type\":\"vxlan\"}}" "$ETCD_SERVER/v2/keys/coreos.com/network/config"
 ```
-
-After configuring flannel, we should restart it for our changes to take effect. Note that this will also restart the docker daemon and could impact running containers.
-
-```sh
-$ sudo systemctl start flanneld
-$ sudo systemctl enable flanneld
+ssh -A core@34.209.11.147
 ```
 
-### Start kubelet
+The important piece of software running is the Kubernetes machine agent, called the kubelet. The  kubelet talks to the Kubernetes API server to:
+* receive tasks to start/stop
+* report back node health, resource usage and metadata
+* Stream back real-time data like logs from applications
 
-Now that everything is configured, we can start the kubelet, which will also start the Pod manifests for the API server, the controller manager, proxy and scheduler.
-
-```sh
-$ sudo systemctl start kubelet
 ```
-
-Ensure that the kubelet will start after a reboot:
-
-```sh
-$ sudo systemctl enable kubelet
-Created symlink from /etc/systemd/system/multi-user.target.wants/kubelet.service to /etc/systemd/system/kubelet.service.
+$ systemctl cat kubelet
 ```
 
-### Basic Health Checks
+A few configuration flags determine important parts of the configuration:
 
-First, we need to make sure the Kubernetes API is available (this could take a few minutes after starting the kubelet.service)
+|Flag   	|Description   	|
+|---	|---	|
+|--kubeconfig   	|A path to a kubeconfig file on disk. This is the same format that was configured above for kubectl, but with different permissions. This is placed on disk by the Tectonic installer.   	|
```

Comment:
Should stick the flags into code blocks. The formatting is breaking as is.

Ex) ``--`` and ``kubeconfig`` are being put on different lines